api/envoy/data/tap/v2alpha/wrapper.proto

```diff
@@ -13,9 +15,38 @@ option java_package = "io.envoyproxy.envoy.data.tap.v2alpha";
 
 // [#protodoc-title: Tap data wrappers]
 
+// [#not-implemented-hide:] Information about the endpoints involved in the tap.
+message Endpoints {
```

Comment:
Can we split this out into a different change? Meaning, I think this is orthogonal to TapDS and applies to all tap types. This will allow us to get streaming in first and then we can add additional data?

Reply:
yes, it is indeed orthogonal. i will remove this and re-introduce later on.
for reference, the goal is to allow higher-level correlations. for example, allow matching the destination of a tap to a specific pod in a kube cluster.